node-manager/src/node.rs

```diff
@@ -891,11 +889,11 @@ pub(crate) async fn connect_peer(
     // first make a connection to the node
     debug!("making connection to peer: {:?}", peer_connection_info);
     let (mut descriptor, socket_addr) = match peer_connection_info.connection_type {
-        ConnectionType::Tcp(t) => {
+        ConnectionType::Tcp(_) => {
             let proxy = Proxy::new(websocket_proxy_addr, peer_connection_info.clone()).await?;
             (
                 WsSocketDescriptor::Tcp(WsTcpSocketDescriptor::new(Arc::new(proxy))),
-                Some(get_net_addr_from_socket(t)),
+                None,
```

Comment:
I think `None` is okay but I wonder how `peer_manager.new_outbound_connection` is using it in the first place?

Maybe instead of strictly passing `None`, you could make a `try_get_net_addr_from_string` method that returns Some or None, depending on if it's a socket address or not.

Reply:
Good idea, done